internal/datastore/crdb/readwrite.go

```diff
@@ -430,7 +431,11 @@ func (rwt *crdbReadWriteTXN) DeleteRelationships(ctx context.Context, filter *v1
 	}
 
 	rwt.relCountChange -= modified.RowsAffected()
-	if delLimit > 0 && uint64(modified.RowsAffected()) == delLimit {
+	rowsAffected, err := safecast.ToUint64(modified.RowsAffected())
+	if err != nil {
+		return false, spiceerrors.MustBugf("could not cast RowsAffected to uint64")
```

Comment:
always wrap the error when using MustBugf

Reply:
In fact, we should probably add a custom linter to ensure this at some point. Might be worth filing an issue